services/core/tests/blockGenerator/generateMockData.js

```diff
@@ -44,10 +44,10 @@ const blockData = {
 		function: () => 1,
 	},
 	timestamp: {
-		function: () => Math.floor(Date.now() / 10) * 10,
+		function: () => Math.floor(Date.now() / 1000),
 	},
 	generatorAddress: {
-		function: () => `${Math.floor(Math.random() * 10 ** 20)}L`,
+		function: () => `${Math.floor(Math.random() * 10 ** 19)}L`,
```

Comment:
Please write a comment that we don't cover the whole range of the address system here.

Reply:
We are covering the entire range of the address system. A valid Lisk address is a maximum of 20 characters long with the suffix `L` included.